app/src/main/res/layout/activity_contest_details.xml

```diff
@@ -1,11 +1,14 @@
 <?xml version="1.0" encoding="utf-8"?>
-<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     tools:context="com.iitp.njack.iitp_connect.CodingCalendar.Activities.ContestDetailsActivity">
 
+    <android.support.constraint.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
     <ImageView
```

Comment:
Indentation is off here. ```ImageView``` and all the elements below are actually children of the ```ConstraintLayout```. Please indent it :smile:

Reply:
Sorry :smile: 
Fixing it :+1:

Reply:
Nothing to be sorry about. It is always good to not format the code in the PR atleast before the reviewer reviews it. Otherwise it leads to a lot of changes which makes it hard to review the actual changes.